docs/conf.py

```diff
@@ -28,15 +28,36 @@
 import datetime
 import os
 import sys
+from pathlib import Path
 
-# Get configuration information from setup.cfg
-from configparser import ConfigParser
+if sys.version_info < (3, 11):
+    import tomli as tomllib
+else:
+    import tomllib
 
-sys.path.insert(1, '..')
+sys.path.insert(0, os.path.abspath('../'))
```

Comment:
Perhaps a silly question. But might this be at fault for the failed RTD docs? If the build process produces then installs a wheel, perhaps the source files do not contain the c extension and adding the source to the path would cause it to take precedence over the installed version. I'm not certain and also not sure how to test this locally.